ui/src/api/beamline.js

```diff
@@ -14,16 +14,22 @@ export function sendPrepareBeamlineForNewSample() {
   return endpoint.put(undefined, '/prepare_beamline').res();
 }
 
-export function sendExecuteCommand(obj, name, args) {
-  return endpoint.post(args, `/${obj}/command/${name}`).res();
+export function sendExecuteCommand(object_type, object_id, command, args) {
```

Comment:
Parameters need to be camel-cased: `objectType`, `objectId`

Reply:
Oh yes :)